.travis.yml

```diff
@@ -11,12 +11,17 @@ matrix:
     - node_js: 8 # Version used to deploy to npm registry
       env: BUILD_ENV=production
 
-cache:
-  directories:
-    - ~/.npm
+cache: yarn
+
+before_install:
+  - yarn global add greenkeeper-lockfile@1
```

Comment:
`@1` for good measure?

Reply:
That's what greenkeeper recommends. And yeah, good idea since we don't know the exact version.